tests/expectations.json

```diff
@@ -20722,7 +20722,6 @@
   "language/expressions/new/spread-sngl-literal.js": "CRASH",
   "language/expressions/new/spread-sngl-obj-ident.js": "CRASH",
   "language/expressions/object/11.1.5-0-1.js": "PASS",
```

Comment:
```suggestion
```

Somehow this passing test got added to the expectations again. Since the original failure was caused by this eval bug, it makes sense to remove it here as well.

Reply:
Fuuck :D